app/models/solidus_paypal_commerce_platform/payment_method.rb

```diff
@@ -69,9 +69,11 @@ def javascript_sdk_url(order: nil, currency: nil)
         intent: auto_capture ? "capture" : "authorize",
         commit: commit_immediately ? "false" : "true",
         components: options[:display_credit_messaging] ? "buttons,messages" : "buttons",
-        currency: currency
+        currency: currency,
```

Comment:
I think the comma here is a typo :stuck_out_tongue:

Reply:
removed and force-pushed